base_tier_validation/views/res_config_settings_views.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="res_config_settings_view_form_budget" model="ir.ui.view">
+        <field name="name">res.config.settings.view.form.budget</field>
+        <field name="model">res.config.settings</field>
+        <field name="priority" eval="25" />
+        <field name="inherit_id" ref="base_setup.res_config_settings_view_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//div[@id='companies']" position="inside">
+                <div id="base_tier_validation">
+                    <h2>Tier Validation</h2>
+                    <div
+                        class="row mt16 o_settings_container"
+                        name="base_tier_validation_option_setting_container"
+                    >
+                        <div
+                            class="col-12 col-lg-6 o_setting_box"
+                            id="module_base_tier_validation_formula"
```

Comment:
All these modules are not yet migrated, and some of them have not even been to 15 yet, so offering these options here seems misleading. Maybe these options should be commented out here and reenabled in each of these modules' migration PRs?